File: part2/ex2list.ts
import {Empleado} from "../part1/ex1class";
let empleados = new Array<Empleado>();

empleados.push(new Empleado('Marina', 600));
empleados.push(new Empleado('Alfredo', 800));

for (let emp of empleados) {
    console.log(emp.getNombre());
}

empleados.forEach(emp => console.log(emp));

File: part2/Ex2list.java
import part1.*;

public class Ex2list {

    public static void main(String [] args) {
        List<Empleado> empleados = new ArrayList<>();

        empleados.add(new Empleado('Marina', 600));
        empleados.add(new Empleado('Alfredo', 800));

        for (Empleado emp: empleados) {
            System.out.println(emp.getNombre());
        }

        empleados.forEach(emp -> {
            System.out.println(emp);
        });
    }
}